### ISSUE TITLE
Trigger on startup

### ISSUE THREAD

**Issue by milosuniver** [eyalroz/removedupes#89]
Is there any easy way to trigger addon on every Thunderbird startup?
I tried tinkering with the code, but I couldn't get it to work on my own.

EDIT: I mean with default and pre-set settings. Like an "auto-mode".

**Comment by milosuniver** [comment]
My question was aimed more at a case where instead of right-clicking on a folder manually and having the addon do its magic, it would perform exactly what it does now, **only once, on startup, and then stop**.

Example now:
- Right-click on inbox
- Remove duplicates 
- Opens up review window
- Click OK, duplicates removed based on the existing settings

Example from my question:
- Open thunderbird
- The addon runs based on existing settings (all folders included, the review window is skipped, everything is already confirmed for first copy for example)
- Duplicates of already recieved emails removed
- After this a manual trigger would be necesarry, or restarting thunderbird (I didn't mean for the addon to be running live)

I am sorry if I am bothering you with this, I didn't mean to pester you with this, I'm just following up to clarify what I meant.
Thank you for your time in any case.

**Comment by eyalroz** [comment]
> only once, on startup

I'm not sure I understand. Only once, or on every startup? TB starts up many times. Your example seems to suggest you want the option for this to run on every startup.

While that may seem convenient, it is _way_ too dangerous. People will have messages deleted which they did not expect - and they won't even know that happened. Some people will change the comparison settings to something too relaxed and will forget the add-on is scheduled to run on startup. No, this would really be a bad idea.

(It's also expensive enough to not be a reasonable thing to do on every startup - once you have a large set of mailboxes, with IMAP etc.)
